docs/gpa.md

```diff
@@ -107,10 +108,10 @@ The addition of `period` to the `Milestone` building block is [under discussion]
         <td markdown=1>
 
 * Use 'open', 'selective' or 'limited' for *the procurement method that will be used* in `tender/procurementMethod`
-* If *it will involve negotiation*, add 'negotiated' to `tender/procurementMethodModalities`
-* If *it will involve electronic auction*, add 'electronicAuction' to `tender/procurementMethodModalities`
+* If *it will involve negotiation*, add 'negotiated' to `tender/procurementMethodDetails`
```

Comment:
Use double quotes for text (single quotes for codes).

For the EU, we have one case of setting it to "Competitive procedure with negotiation", so I think "negotiated" here is fine.

Reply:
Done